app/views/docs/functions.phtml

```diff
@@ -7,920 +7,200 @@ $runtimes = $this->getParam('runtimes', []);
-</div>
-
-<p>Give your function a name and choose your runtime. This will create a new starter function in the current directory and also add it to your <code>appwrite.json</code> file. Go ahead and deploy your function using :</p>
-
-<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
-    <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite deploy function</code></pre>
-</div>
-
-<p>You can now head over to your Appwrite Dashboard, navigate to the Function settings and execute your function. You can find the status of your execution under the <b>Logs</b> tab. </p>
-
-<p>Feel free to modify and play around with the starter code and use the <code>appwrite deploy</code> command to instantly deploy your changes to the Appwrite server. </p>
-
-<p> The following sections will dive deeper into some more terminology and advanced concepts which can be useful when writing your function from scratch.</p>
-
-<h2><a href="/docs/functions#writingYourOwnFunction" id="writingYourOwnFunction">Writing your own Function</a></h2>
-
-<p>When writing your own Appwrite Function, you must export the code in certain ways depending on the language. This varies between languages so refer to the examples below.</p>
+<p>
+    Appwrite Functions let you extend Appwrite by adding your own code and logic. 
+    You can think of them as code snippets that are triggered by server events, webhooks, scheduled executions, or user invokation.
+    Each function will have their own URL, execute in their own isolated container, and have their own configurable environment variables and permissions.
+    With these features, Appwrite Functions unlock limitless potential to expand Appwrite's capabilities with custom logic and integrations.
+</p>
+
+<h2><a href="#gettingStarted" id="gettingStarted">Getting Started</a></h2>
+<p>
+    Appwrite Functions unlock limitless possibilities, but it's simple to get started. You can deploy your first function and execute it in minutes.
+</p>
 
 <ul class="phases clear" data-ui-phases>
     <li>
         <h3>Node.js</h3>
         <div>
         <div class="ide margin-top-small" data-lang="javascript" data-lang-label="JavaScript">
-            <pre class="line-numbers"><code class="prism language-js" data-prism>module.exports = async (req, res) => {
-  const payload =
-    req.payload ||
-    'No payload provided. Add custom data when executing function.';
-
-  const secretKey =
-    req.variables.SECRET_KEY ||
-    'SECRET_KEY variable not found. You can set it in Function settings.';
-
-  const randomNumber = Math.random();
-
-  const trigger = req.variables.APPWRITE_FUNCTION_TRIGGER;
-
-  res.json({
-    message: 'Hello from Appwrite!',
-    payload,
-    secretKey,
-    randomNumber,
-    trigger,
+            <pre class="line-numbers"><code class="prism language-js" data-prism>export default async ({ res }) => {
```

Comment:
Here you go! @loks0n, something to start, lets translate this example to all the other runtimes.

Reply:
Done :)